src/ToDoHeader/ToDoHeader.css

```
@@ -0,0 +1,28 @@
+.ToDoHeader {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 5px;
+  background-color: var(--blue-color-4);
+  border-bottom: var(--spacing-0) solid var(--brown-color-2);
+  min-height: 100px;
+
+  --subtitle-size: var(--spacing-3);
+}
+
+.ToDoHeader-title {
+  margin: var(--spacing-3) 0;
+  color: var(--green-color-1);
+}
+
+.ToDoHeader-subtitle {
+  margin: 0 0 var(--subtitle-size) 0;
```

If there's only margin or padding on one side, we often specify that side explicitly.
```suggestion
  margin-bottom: var(--subtitle-size);
```

---

I brought in [`normalize.css`](https://necolas.github.io/normalize.css/) as a CSS reset, but it appears to apply margins to the other sides when I specify only `margin-bottom`.  I may need to reconsider my CSS reset choice.

---

Added the following to `index.css` for a simpler reset to work with:
```
* {
  padding: 0;
  margin: 0;
}
```

And I've updated ToDoHeader.css with your suggestion in 742fa8b